README.md

```diff
@@ -132,6 +132,51 @@ Additionally it will monitor etcd for new members of the network and adjust the
 
 After flannel has acquired the subnet and configured backend, it will write out an environment variable file (`/run/flannel/subnet.env` by default) with subnet address and MTU that it supports.
 
+## Client/Server mode (EXPERIMENTAL)
```

Comment:
If I have a client/server set up can I also run a normal flannel daemon in the same overlay?